app/handlers/newflow/educator_signup/sheerid_webhook.rb

```diff
@@ -16,7 +16,7 @@ def authorized?
       def handle
         verification_id = params.fetch(VERIFICATION_ID_PARAM_NAME, nil) || params.fetch(REQUEST_ID_PARAM)
         outputs.verification_id = verification_id
-        run(ProcessSheeridWebhookRequest, verification_id: verification_id)
+        ProcessSheeridWebhookRequest.perform_later(verification_id: verification_id)
```

Comment:
does perform later send this to a background job?

Reply:
@mwharrison Yes.

Reply:
One advantage is that jobs retry if/when they fail